src/main/webapp/components/InternationalCohorts.js

```diff
@@ -38,7 +38,7 @@ export const InternationalCohorts = hh(class InternationalCohorts extends Compon
       yesOutput: 3,
       noOutput: DETERMINATION.EXIT,
       answer: null,
-      progress: 17,
+      progress: 20,
```

Comment:
nit: not your code, so does not have to be addressed in this PR. This really should be a calculation, not a hard number. Future changes would only need to alter the formula instead of each particular progress level.

Reply:
Agree.

Reply:
https://broadinstitute.atlassian.net/browse/BTRX-730 <- done